pages/common/valac.md

```diff
@@ -0,0 +1,17 @@
+# valac
+
+> Vala code compiler.
+> Tutorial: <https://wiki.gnome.org/Projects/Vala/Tutorial>.
+> Doc homepage: <https://valadoc.org/>.
```

Comment:
I think this should be the same as above, unless there is a direct link to the `valac` docs?
```suggestion
> More information: <https://wiki.gnome.org/Projects/Vala>.
```